src/libraries/System.Diagnostics.Process/tests/ProcessStandardConsoleTests.cs

```diff
@@ -42,20 +44,22 @@ public void TestChangesInConsoleEncoding()
 
             try
             {
-                // Don't test this on Windows Nano, Windows Nano only supports UTF8.
-                if (File.Exists(Path.Combine(Environment.GetEnvironmentVariable("windir"), "regedit.exe")))
-                {
-                    Interop.SetConsoleCP(s_ConsoleEncoding);
-                    Interop.SetConsoleOutputCP(s_ConsoleEncoding);
-
-                    run(s_ConsoleEncoding);
-                }
+                Interop.SetConsoleCP(ConsoleEncoding);
+                Interop.SetConsoleOutputCP(ConsoleEncoding);
+
+                RunWithExpectedCodePage(ConsoleEncoding);
             }
             finally
             {
                 Interop.SetConsoleCP(inputEncoding);
                 Interop.SetConsoleOutputCP(outputEncoding);
             }
         }
+
+        private static bool IsRunningInWindowsContainer()
```

Comment:
such method already exists in the `PlatformDetection` utility class:

https://github.com/dotnet/runtime/blob/161f53c6175f6a0908f30d7df934eb8118f09598/src/libraries/Common/tests/TestUtilities/System/PlatformDetection.cs#L224

Could you please reuse it?

Reply:
Good call, addressed.